third_party/gfootball_engine/src/systems/graphics/rendering/opengl_renderer3d.cpp

```diff
@@ -1434,6 +1461,7 @@ void OpenGLRenderer3D::SetLight(const Vector3 &position, const Vector3 &color,
   mapping.glLightfv(GL_LIGHT0, GL_SPECULAR, specular);
 
   mapping.glEnable(GL_LIGHT0);
+   */
```

Comment:
Again, the method is never called and `GL_LIGHTING` is never enabled in the first place because it's rendered with shaders